coalib/bearlib/aspects/Formatting.py

```diff
@@ -0,0 +1,359 @@
+        only when necessary.
+        """
+
+
+@BlankLine.subaspect
+class BlankLineAfterDeclaration:
+    """
+    Those found after declarations.
+    """
+    class docs:
+        example = """
+        int main ()
+        {
+          int a;
+          float b;
+
+          scanf("%d%f", &a, &b);
+          printf("a = %d and b = %f", a, b);
+          return 0;
+        }
+        """
+        example_language = 'C'
+        importance_reason = """
+        Having a specific and reasonable number of blank lines after every
+        block of declarations improves on the readability of the code.
+        """
+        fix_suggestions = """
+        `BlankLintAfterDeclaration` issues can be fixed specifying (and of
+        course using) a reasonable number of blank lines to use after block
+        declaration.
+        """
+    blank_lines_after_declarations = Taste[int](
+        'Represents the number of blank lines after declarations',
+        (0, 1, 2), default=0)
+
+
+@BlankLine.subaspect
+class BlankLineAfterProcedure:
+    """
+    Those found after procedures or functions.
+    """
+    class docs:
+        example = """
+        void proc(void){
+            printf("this does nothing");
+        }
+
+        int add(float a, float b){
+            return a + b;
+        }
+        """
+        example_language = 'C'
+        importance_reason = """
+        Having a specific and reasonable number of blank lines after every
+        procedures improves on the readability of the code.
+        """
+        fix_suggestions = """
+        `BlankLintAfterProcedure` issues can be fixed specifying (and of
+        course using) a reasonable number of blank lines to use after
+        procedures' definition.
+        """
+    blank_lines_after_procedures = Taste[int](
+        'Represents the number of blank lines to use after a procedure or a'
+        ' function', (0, 1, 2), default=1)
+
+
+@BlankLine.subaspect
+class BlankLineAfterClass:
+    """
+    Those found after classes' definitions.
+    """
+    class docs:
+        example = """
+        class SomeClass:
+            def __init__(self):
+                raise RuntimeError('Never instantiate this class')
+
+
+        def func():
+            pass
+        """
+        example_language = 'Python 3'
+        importance_reason = """
+        Having a specific number of blank lines after every classes'
+        definitions declarations improves on the readability of the code.
+        """
+        fix_suggestions = """
+        """
+    blank_lines_after_class = Taste[int](
+        'Represents the number of blank lines to use after a class'
+        'definition.', (1, 2), default=2)
+
+
+@BlankLine.subaspect
+class NewlineAtEOF:
+    """
+    Newline character (usually '\\\\n', aka CR) found at the end of file.
+    """
+    class docs:
+        example = """
+        def do_nothing():
+            pass
+        """ + ('\n')
+        example_language = 'Python 3'
```

Comment:
this was correct as just `Python`, as it uses features available and consistent in all Python versions.